src/EppoClient.php

```diff
@@ -42,6 +42,9 @@ class EppoClient
     /** @var PollerInterface */
     private $poller;
 
+    /** @var bool */
+    private $isGracefulMode;
```

Comment:
consider naming something more explicit like `isGracefulHandleExceptions`

Reply:
In the other SDKs are named `isGracefulMode` - I agree you suggestion is more clear. I would like to update them all at the same time to be consistent.